### PR TITLE
feat(bidi): add rtl support

### PR DESCRIPTION
**Pull request by gergelyke**

### CODE REVIEW

File: flowtypes/Range.js.flow [hunk 1]
```diff
@@ -28,6 +28,7 @@ declare interface IProps {
   direction?: string;
   allowOverlap?: boolean;
   disabled?: boolean;
+  rtl: boolean;
```
Comment (chasestarr) [inline flowtypes/Range.js.flow]: should this be optional similar to `disabled` above?

```suggestion
  rtl?: boolean;
```

File: src/Range.tsx [hunk 2]
```diff
@@ -20,6 +20,7 @@ class Range extends React.Component<IProps> {
   static defaultProps = {
     step: 1,
     direction: Direction.Right,
+    rtl: false,
```
Comment (chasestarr) [inline src/Range.tsx]: keyboard controls (left, right arrow) are reversed for rtl examples

**Comment by gergelyke** [comment]
thanks, @chasestarr  - both of them should be fixed

**Comment by nadiia** [comment]
There seem to be something wrong with the thumbs positioning, they're not visible:
Screen Shot 2019-08-16 at 12 40 47 PM

**Comment by gergelyke** [comment]
> There seem to be something wrong with the thumbs positioning, they're not visible:
> Screen Shot 2019-08-16 at 12 40 47 PM

have you triggered the RTL addon?

**Comment by tajo** [comment]
```
+ react-range@1.3.0
```

**Pull request merged**